src/client/Dashboard.types.ts

```diff
@@ -60,4 +60,5 @@ export interface IDeploymentField {
   prSourceBranch?: string;
   mergedByName?: string;
   mergedByImageURL?: string;
+  manifestCommitId?: string;
```

Comment:
@samiyaakhtar any reason this wasn't defined before?

Reply:
It was missing from the type because the `IDeploymentField` was defined inline, now that I moved it to a separate function, the linter enforced it to be defined in the interface.